src/app/Console/Commands/Install.php

```diff
@@ -26,45 +31,191 @@ class Install extends Command
+        $userModel = new $userClass();
+
+        // Count current users
+        $currentUsers = $userModel->count();
+
+        $this->newLine();
+        $this->infoBlock('Creating an admin:', 'Step 2');
+        $this->note('Quickly jump in your admin panel, using the email & password you choose here.');
+        $this->note('Currently there '.trans_choice("{0} are <fg=blue>no users</>|{1} is <fg=blue>1 user</>|[2,*] are <fg=blue>$currentUsers users</>", $currentUsers).' in the database.');
+
+        $total = 0;
+        while ($this->confirm(' Add '.($total ? 'another' : 'an').' admin user?')) {
+            $name = $this->ask(' Name');
+            $mail = $this->ask(" {$name}'s email");
+            $pass = $this->secret(" {$name}'s password");
+
+            try {
+                $userModel->insert([
+                    'name' => $name,
+                    'email' => $mail,
+                    'password' => bcrypt($pass),
+                    'created_at' => Carbon::now(),
+                    'updated_at' => Carbon::now(),
+                ]);
+
+                $this->deleteLines(12);
+                $this->progressBlock('Adding admin user');
+                $this->closeProgressBlock();
+                $this->note($name);
+                $this->note($mail);
+
+                $total++;
+            } catch (\Throwable$e) {
+                $this->errorBlock($e->getMessage());
+            }
+        }
+
+        $this->deleteLines(3);
+
+        if (! $total) {
+            $this->deleteLines();
+            $this->note('Skipping creating an admin user.');
+            $this->newLine();
+        }
+    }
+
+    private function isEveryAddonInstalled()
+    {
+        return collect($this->addons)->every(function ($addon) {
+            return file_exists($addon->path);
+        });
+    }
+
+    private function updateAddonsStatus()
+    {
+        $this->addons = $this->addons->each(function (&$addon) {
+            $isInstalled = file_exists($addon->path);
+            $addon->status = $isInstalled ? 'installed' : 'not installed';
+            $addon->statusColor = $isInstalled ? 'green' : 'yellow';
+        });
+    }
+
+    private function installAddons()
+    {
+        // map the addons
+        $this->addons = collect($this->addons)
+            ->map(function ($class) {
+                return (object) $class::$addon;
+            });
+
+        // set addons current status (installed / not installed)
+        $this->updateAddonsStatus();
+
+        // if all addons are installed do nothing
+        if ($this->isEveryAddonInstalled()) {
+            return;
+        }
+
+        $this->infoBlock('Installing premium Backpack add-ons:', 'Step 3');
+        $this->note('Add tons of features and functionality to your admin panel, using our paid add-ons.');
+        $this->note('For more information, payment and access please visit https://backpackforlaravel.com/pricing');
```

Comment:
Do your magic trick with the colors here in this link too @promatik 🌈

Reply:
Yes please!

Reply:
Done 👌